password_security/models/res_users.py

```diff
@@ -92,24 +92,41 @@ def password_match_message(self):
         company_id = self.company_id
         message = []
         if company_id.password_lower:
-            message.append('\n* ' + 'Lowercase letter (At least ' + str(
-                company_id.password_lower) + ' character)')
+            message.append(
+                _(
+                    "\n* Lowercase letter (at least %s characters)"
```

Comment:
I think you should use a multiple options word instead of a plural or singular only.
Same with all the other **character** words.
```suggestion
                    "\n* Lowercase letter (at least %s character(s))"
```

Reply:
> @TheRaphael0000 I think you should use a multiple options word instead of a plural or singular only. Same with all the other **"characters"** words.

Even better, this guideline show how to handle plural forms:
 
https://www.odoo.com/documentation/14.0/developer/misc/i18n/translations.html#plural